DOCKER.md

```diff
@@ -13,3 +13,17 @@ ADMIN_EMAILS=
 SENDGRID_USERNAME=
 SENDGRID_PASSWORD=
 ```
+
+If you would like to use [Amazon SES](https://aws.amazon.com/ses/) instead to send emails, you'll need a different set of environmental variables.
+
+```sh
+DATABASE_URL=
+SECRET_KEY_BASE=
+ADMIN_EMAILS=
```

Comment:
True — I included them because the SendGrid example also had them, and I wanted it to super explicit which environmental variables are required in both cases.

It could instead be re-written to say:
1) You always need these
2) And then pick one of these sets

Although I personally do like that you can copy-paste the entire template of what you need in one go.